test/python/test_util.py

```diff
@@ -21,11 +22,11 @@
 class TestUtil(QiskitTestCase):
     """Tests for qiskit/_util.py"""
 
+    @skipIf(not hasattr(os, "sched_getaffinity"), "ched_getaffinity is only available on linux")
```

Comment:
With the changes suggested above is this needed?

Reply:
"Mock" was giving an error about sched_getaffinity not existing, so I assumed, that it still tries to access the method, to setup the mock-method. This is why I added the skip-decorator.